js/views/onboardingModal.js

```diff
@@ -351,6 +351,10 @@ module.exports = baseModal.extend({
                 console.log(errorThrown);
               }
             });
+          } else {
```

Comment:
If I remember right, it was when you set up two nodes on the same ip, but with different ports. The server connection works, but saving to settings doesn't, and in this case when the save to settings failed you were stuck.

Since the situation where this occurs is if you connect to a server you shouldn't, the only way out of it is to show the connect dialog. Unfortunately the client can't tell that's the reason settings failed, so we can't show a customized error message, which would be ideal.